modules/addtask.js

```diff
@@ -0,0 +1,23 @@
+const todoList = document.getElementById('todoList');
+
+module.exports = function addtask(boo) {
```

Comment:
- [ ] I will strongly recommend you use a descriptive name for your addtask parameter `boo`. Kindly change `boo` to something that is easily relatable to the feature you are working on 👍🏽